gradle.properties

```diff
@@ -1,7 +1,7 @@
 # Main properties
 group=com.truelayer
 archivesBaseName=truelayer-java
-version=9.0.0
+version=9.1.0
```

Comment:
This is probably going to be a major bump as we're changing this [public interface](https://github.com/TrueLayer/truelayer-java/pull/213/files#diff-8631178e3e985813976b00c114f1e1c7cf0cba1378630ab8bfe64f12fab24082R7)